frontend/src/app/patients/personSchema.ts

```diff
@@ -84,7 +84,16 @@ export function areValidPhoneNumbers(phoneNumbers: any) {
     return phoneNumberIsValid(phoneNumber.number);
   });
 }
-
+const isValidDate = (date: any) => {
+  const [year] = date.split("-");
+  if (parseInt(year) < 1900) {
+    return false;
+  }
+  if (new Date(date) > new Date()) {
+    return false;
+  }
+  return true;
```

Comment:
I think this function will return `true` if `date` is an empty string (or a single character space, and probably some other edge cases). Consider using something like [moment's isValid method](https://momentjs.com/docs/#/parsing/is-valid/) to check if the provided string is even a valid date in the first place.

If you end up using `moment`, we could probably use some of their other methods to test the date range we want here rather than rolling our own logic

Reply:
@nickscialli-usds I've replaced this custom parsing with a usage of `moment`; would love a re-review!